### DISCUSSION TITLE
Take out password or obfuscate it on NatsException

### DISCUSSION THREAD

**Discussion by reshma-raveendran** [nats-io/nats.net#699] [General]
I'm using nats 2.2.3 and noticing that on nats connection exceptions the connection string (including the password) is part of the message. Is there a setting to hide or not include the password in the message. For e,g 1 of the natsexception is can not connect uris: nats://admin:password@hostname:port

**Comment by wallyqs** [comment]
This would be a client related setting, which is the client that you are using? Java?

**Reply by reshma-raveendran** [reply]
Its a .net client

**Reply by wallyqs** [reply]
ok need to fix that in the .NET client since that is what is generating the exception

**Reply by reshma-raveendran** [reply]
Thanks, is there anything further required to address this from my end?

**Reply by wallyqs** [reply]
I moved it the issue to the nats.net client, is that the one you are using? there is a nats.net.v1 too

**Reply by reshma-raveendran** [reply]
This looks right, thanks

**Comment by mtmk** [comment]
@reshma-raveendran I believe this was fixed in https://github.com/nats-io/nats.net/releases/tag/v2.5.3

Could you confirm 2.5.3 works for you?

**Reply by reshma-raveendran** [reply]
Thank you, yes it does work on 2.5.3!
